ml-agents-envs/mlagents/envs/tests/test_sampler_class.py

```diff
@@ -0,0 +1,154 @@
+from math import isclose
+import pytest
+
+from mlagents.envs.sampler_class import SamplerManager
+from mlagents.envs.sampler_class import UniformSampler, MultiRangeUniformSampler
+from mlagents.envs.exception import UnityException
+
+
+def basic_3Dball_sampler():
```

Comment:
Any reason these need to be functions? Why not just put the dict literal in the test function?

Reply:
Just put them as function for cleaner code along with consistency with other test files. Nothing special otherwise.